package.json

```diff
@@ -10,9 +10,13 @@
     "javascript"
   ],
   "author": "leilers@vistaprint.com",
-  "license": "Apache 2.0",
```

Comment:
I only changed this because  powershell gave the following error 
![image](https://cloud.githubusercontent.com/assets/1136514/12988787/6fea25e8-d0d0-11e5-8896-6ad5a65d61ec.png)